frontend/src/apiMocks/handlers.ts

```diff
@@ -0,0 +1,237 @@
+import { rest, RestHandler, RestRequest } from "msw";
+import { CustomAliasData, RandomAliasData } from "../hooks/api/aliases";
+import { ProfileData } from "../hooks/api/profile";
+import { RuntimeData } from "../hooks/api/runtimeData";
+import { mockIds, domainaddresses, profiles, relayaddresses } from "./mockData";
+
+export function getHandlers(
+  defaultMockId: null | typeof mockIds[number] = null
+): RestHandler[] {
+  const handlers: RestHandler[] = [];
```

Comment:
3. Here we define the handlers. They're loaded by the Service Worker, and respond to the listed HTTP requests.